std/typecons.d

```diff
@@ -799,35 +799,73 @@ if (distinctFieldNames!(Specs))
          *           for comparison between `Tuple`s.
          *
          * Returns:
-         * For any values `v1` on the right-hand side and `v2` on the
-         * left-hand side:
+         * For values `v1` on the left-hand side and `v2` on the right-hand side:
+         *
+         * 0 if the expression `v1 == v2` is true for all members, or else for
+         * the first member, where this is not true:
```

Comment:
I would rephrase this to:

"For any values `v1` contained by the left-hand side tuple and any values `v2` contained by the right-hand side:

0 if `v1 == v2` for all members or the following value for the first position were the mentioned criteria is not satisfied:"

Reply:
done - thanks